src/Microsoft.Azure.WebJobs.ServiceBus/EventHubs/EventHubTriggerAttributeBindingProvider.cs

```diff
@@ -62,7 +62,7 @@ public Task<ITriggerBinding> TryCreateAsync(TriggerBindingProviderContext contex
             Func<ListenerFactoryContext, bool, Task<IListener>> createListener =
              (factoryContext, singleDispatch) =>
              {
-                 IListener listener = new EventHubListener(factoryContext.Executor, eventHostListener, options, singleDispatch);
+                 IListener listener = new EventHubListener(factoryContext.Executor, eventHostListener, options, singleDispatch, _eventHubConfig.BatchCheckpointFrequency);
```

Comment:
Why not just pass in the entire config object to the listener, so we don't have to modify this param list each time we add a new knob?